doc/waarp-r66/source/configuration-details/detail-tasks.rst

```diff
@@ -2,6 +2,83 @@
 Type de Task
 ############
 
+Format général d'une tâche
+--------------------------
+
+Une tâche est définie selon un format unifié XML :
+
+.. code-block:: xml
+
+  <tasks>
+   <task>
+    <type>NAME</type>
+    <path>path</path>
+    <delay>x</delay>
+    <rank>n</rank>
```

Comment:
It is the first time I see this `<rank>` tag...
What does it do?
it is not documented below, and it is not used in any task...

Reply:
Added an explanation: this is an optional element (long time ago) which allows, mainly for test and validation purpose, to specify the order of execution, while it is implicit if not specified in written order.